resources/views/site/blocks/singleCheckbox.blade.php

```diff
@@ -0,0 +1,12 @@
+@if (!session('checkboxConsent' . str_replace(' ', '-', strtolower(trim($block->translatedinput('title'))))))
```

Comment:
```suggestion
@if (!session('checkboxConsent' . Str::slug($block->translatedinput('title')))
```

Reply:
Thank you for the tip, we simplified the code